web/static/css/prom_console.css

```diff
@@ -26,7 +30,7 @@
   pointer-events: none;
   cursor: default;
 }
-.prom_lhs_menu_selected {
+li.prom_lhs_menu_selected {
   background: #555555;
   background-clip: padding-box;
 }
```

Comment:
GitHub doesn't let me comment on the correct line because it's not part of the diff, but if you change the selectors in this part:

```
.prom_lhs_menu span:hover,
.prom_lhs_menu a:hover {
  background: #666666;
}
```

to this, then the hovered and selected box sizes are the same:

```
.prom_lhs_menu li:hover {
  background: #666666;
}
```